packages/patternfly-3/patternfly-react/src/components/HorizontalNav/HorizontalNavMenuItem.js

```diff
@@ -35,7 +35,7 @@ const HorizontalNavMenuItem = props => {
 HorizontalNavMenuItem.propTypes = {
   children: PropTypes.node,
   onItemClick: PropTypes.object,
-  title: PropTypes.string,
+  title: PropTypes.node,
```

Comment:
While node works for both, we typically have used:
```suggestion
  title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```
to be more explicit

Reply:
Not sure that it's required node can be anything that can be rendered https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes

Reply:
Understood, just looking for consistency and explicitly noting in the docs.

Reply:
Guys, I do not care ;-)

I am seeing both variants in places. Tell me if I need to change it to get the PR merged or not.

Yes - change / No - leave it.

That's what I care about :-D Because w/o that I cannot finish the PRs above.

Reply:
Thx!